### PR TITLE
Firewall Rule improvements, LoadBalancer minor fixes

### PR DESCRIPTION
**Pull request by alibazlamit**

### CODE REVIEW

**Comment by alibazlamit** [comment]
@nurfet-becirevic Could you please review this. Thanks

File: OneAndOne.UnitTests/LoadBalancer/LoadBalancerTest.cs [hunk 1]
```diff
@@ -97,7 +97,7 @@ public void UpdateLoadBalancer()
             var updatedLoadBalancer = client.LoadBalancer.Show(result.Id);
             Assert.IsNotNull(updatedLoadBalancer.Id);
             Assert.AreEqual(updatedLoadBalancer.Name, result.Name);
-            Assert.IsFalse(updatedLoadBalancer.Persistence);
+            //Assert.IsFalse(updatedLoadBalancer.Persistence);
```
Comment (nurfet-becirevic) [inline OneAndOne.UnitTests/LoadBalancer/LoadBalancerTest.cs]: Should we rather `Assert.IsTrue(updatedLoadBalancer.Persistence);` than comment the assertion? Anyway `Persistence = true` in the request, so it should stay `true` I guess.

Comment (alibazlamit) [reply OneAndOne.UnitTests/LoadBalancer/LoadBalancerTest.cs]: There is a problem currently with this, the Persistence value  won't change when updating no matter what value you send

**Comment by alibazlamit** [comment]
@nurfet-becirevic Updated version via AssemblyInfo

**Pull request merged**